### PR TITLE
TRACING-4559: clarify replication factor

### PR DESCRIPTION
**Pull request by iblancasa**
Version(s):  4.18, 4.17, 4.16, 4.15, 4.14, 4.13, 4.12
Issue: [TRACING-4559](https://issues.redhat.com//browse/TRACING-4559)

Link to docs preview:  https://82367--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-configuring.html

QE review:
- [x] QE has approved this change.

Additional information: this can be added to the latest version of the documentation since it applies to the current RHOSDT version too.

### CODE REVIEW

**Comment by iblancasa** [comment]
@max-cx @IshwarKanse

File: modules/distr-tracing-tempo-config-default.adoc [hunk 1]
```diff
@@ -45,7 +45,7 @@ managementState: managed # <21>
 <5> Contains all of the configuration parameters of the TempoStack instance. When a common definition for all Tempo components is required, define it in the `spec` section. When the definition relates to an individual component, place it in the `spec.template.<component>` section.
 <6> Storage is specified at instance deployment. See the installation page for information about storage options for the instance.
 <7> Defines the compute resources for the Tempo container.
-<8> Configuration for the replication factor.
+<8> Configuration for the replication factor. The replication factor is a configuration setting that determines how many copies of your trace data are stored across different instances of Tempo's backend storage. For example, a replication factor of 3 means each trace is written to three different nodes, providing resilience against up to two node failures.
```
Comment (IshwarKanse) [inline modules/distr-tracing-tempo-config-default.adoc:48]: We also need to change the upstream API docs https://github.com/grafana/tempo-operator/blob/main/docs/spec/tempo.grafana.com_tempostacks.yaml#L57 as the information is misleading.

**Review by IshwarKanse — APPROVED** [review]

**Comment by iblancasa** [comment]
@max-cx this PR is ready for review again

File: modules/distr-tracing-tempo-config-default.adoc [hunk 2]
```diff
@@ -45,7 +45,7 @@ managementState: managed # <21>
 <5> Contains all of the configuration parameters of the TempoStack instance. When a common definition for all Tempo components is required, define it in the `spec` section. When the definition relates to an individual component, place it in the `spec.template.<component>` section.
 <6> Storage is specified at instance deployment. See the installation page for information about storage options for the instance.
 <7> Defines the compute resources for the Tempo container.
-<8> Configuration for the replication factor.
+<8> Configuration for the replication factor. The replication factor is a configuration setting that determines how many ingesters need to acknowledge the data from the distributors before accepting a span.
```
Comment (max-cx) [inline modules/distr-tracing-tempo-config-default.adoc:48]:
```suggestion
<8> Configuration for the replication factor. The replication factor is a configuration setting that determines how many ingesters must acknowledge the data from the distributors before accepting a span.
```

**Review by max-cx — APPROVED** [review]
LGTM.

**Comment by max-cx** [comment]
LGTM

**Comment by max-cx** [comment]
Preview link:

https://82367--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-configuring.html#distr-tracing-tempo-config-default_distr-tracing-tempo-configuring

**Pull request merged**